CONTRIBUTING: add code comment style (#23)
pingcap/community: CONTRIBUTING.md

The coding style suggested by the Golang community is used in TiDB. See the [style doc](https://github.com/golang/go/wiki/CodeReviewComments) for details.

### Code comment style

Follow this style to write TiDB code comment.

For the code comment style of the TiKV project, see [Code Comment Style](https://github.com/tikv/tikv/blob/master/CODE_COMMENT_STYLE.md).

**Format:**

- Use `//` for a single-line comment and trailing comment.
- Use `/* ... */` for a block comment (used only when needed).
- Use **gofmt** to format your code.

> - Place the single-line and block comment above the code it’s annotating.

Reviewer: ```suggestion
- Place the single-line or block comment above the code it's annotating.
```
